db/migrations/0002_alter_skill_bonus.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-12-15 12:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='bonus',
+            field=models.CharField(max_length=255),
```

Comment:
The `bonus` field in the `Skill` model is being altered to a `CharField`, which is incorrect. It should be an `IntegerField` to store numeric values. Please change this field back to `IntegerField` and ensure there is no `max_length` parameter.